compatibility_server/configs.py

```diff
@@ -1,187 +0,0 @@
-# Copyright 2018 Google LLC
```

Comment:
The compatibility server is deployed using docker, and it won't be able to find the symlinks without uploading the ../compatibility_lib/ stuff with the same directory structure unfortunately :(

Reply:
That's a good point - what if compatibility_lib/compatibility_lib/configs.py is the symlink file instead?

Reply:
That's the same, releasing the compatibility_lib to PyPI won't include the files in compatibility_server.

Reply:
hmm I guess they're pretty isolated from each other

Maybe I can add a test to catch any differences instead.

Reply:
SGTM.

Reply:
done